packages/MusicNotation-Scripting.package/MNNoteBuilder.class/class/context.pitch..st

```diff
@@ -0,0 +1,5 @@
+instance creation
+context: aContext pitch: aString
```

Comment:
Ich finde die Methoden ein wenig inkonsistent:
`context: aContext pitches: aCollection` erwartet eine Collection an `MNPitch`es, während `context: aContext pitch: aString` einen `String`erwartet.

Hier würde ich vom Methodennamen erwarten, dass auch eine `MNPitch`erwartet wird... daher vlt. in `context: aContext pitchFrom: aString` umbenennen.